Sources/SwiftGen/main.swift

```diff
@@ -23,21 +24,37 @@ let main = Group {
     }
   }
   $0.group("config", "manage and run configuration files") {
-    $0.addCommand("lint", "Lint the configuration file", ConfigCLI.lint)
-    $0.addCommand("run", "Run commands listed in the configuration file", ConfigCLI.run)
-    $0.addCommand("init", "Create an initial configuration file", ConfigCLI.create)
-    $0.addCommand("doc", "Open the documentation for the configuration file on GitHub", ConfigCLI.doc)
+    $0.addCommand("lint", "lint the configuration file", ConfigCLI.lint)
+    $0.addCommand("run", "run commands listed in the configuration file", ConfigCLI.run)
+    $0.addCommand("init", "create an initial configuration file", ConfigCLI.create)
+    $0.addCommand("doc", "open the documentation for the configuration file on GitHub", ConfigCLI.doc)
   }
 
-  $0.group("templates", "manage custom templates") {
+  $0.group("template", "manage custom templates") {
     $0.addCommand("list", "list bundled and custom templates", TemplatesCLI.list)
     $0.addCommand("which", "print path of a given named template", TemplatesCLI.which)
     $0.addCommand("cat", "print content of a given named template", TemplatesCLI.cat)
+    $0.addCommand("doc", "open the documentation for templates on GitHub", TemplatesCLI.doc)
+  }
+
+  $0.group("run", "run individual parser commands without a config file") {
```

Comment:
Is this the name we want for that subcommand? An alternate solution I thought about was to call that `parse`, so that it reads e.g. `swiftgen parse xcassets --templateName swift5 -o Assets.swift Images.xcassets`… thoughts?

Reply:
"Parse" sounds ok. If I'm not mistaken we *do* already have a `config run` command, so there is a precedent there.